src/Sulu/Bundle/AdminBundle/Resources/js/containers/Navigation/Navigation.js

```diff
@@ -67,9 +68,11 @@ class Navigation extends React.Component<Props> {
     render() {
         const {suluVersion} = this.props;
         const {appVersion} = this.props;
```

Comment:
I know that code hasn't been touched by you, but while you're at it, you could combine the previous two destructurings:
```javascript
const {appVersion, suluVersion} = this.props;
```